tools/gen-intrinsics-specs.py

```diff
@@ -688,19 +758,32 @@ def recurse_print_to_rst(item, section_level_list, headers=__intrinsic_table_hea
     +=============+========================+=======================+==========+===========================+
     |           1 |                      2 |                     3 |        4 |                         5 |
     +-------------+------------------------+-----------------------+----------+---------------------------+
+    >>> print(recurse_print_to_rst(item, ['#','##'], tablefmt='pdf'))
+    <BLANKLINE>
+    # Section 1
+    <BLANKLINE>
+    ## Section 1.1
+    <BLANKLINE>
+    Text for Section 1.1
+    <BLANKLINE>
+    +-------------+------------------------+-----------------------+----------+---------------------------+
+    |   Intrinsic |   Argument preparation |   AArch64 Instruction |   Result |   Supported architectures |
+    +=============+========================+=======================+==========+===========================+
+    |           1 |                      2 |                     3 |        4 |                         5 |
+    +-------------+------------------------+-----------------------+----------+---------------------------+
     """
     key, value = item
     if is_intrinsic_table(item):
-        return "\n" + str(tbl.tabulate(value, headers=headers, tablefmt=tablefmt))
+        return "\n" + str(tbl.tabulate(value, headers=headers, tablefmt=(tablefmt if tablefmt != "pdf" else "grid")))
```

Comment:
This isn't using table_format. I'm using the dictionary as part of the recursive_print_to_rst call in process_db on line 1010, because otherwise it's either a case of the grid format being printed out with the wrong headers or the rst format defaulting to grid. I can optimise this call once we've started removing rst support, though.

Reply:
Wait - aren't you passing the option `--source_syntax` with three possible values? `rst`, `markdown` and `pdf`.

This seems to be be enough to choose  the combinations?
`rst` maps to `rst` and `grid` table format
`markdown` maps to `markdown` syntax and `github` table format
`pdf` maps to `markdown` syntax and `grid` table format.

I guess that part of the confusion comes from the name of the option? In fact the option is not really choosing the source format, it is choosing the workflow we want to generate the sources for. but for each workflow, we can unequivocally determine the two values we need: 1. source format (`rst` vs `markdown`), 2. table format (`github` vs `grid`).

Maybe, because of this, we should change the name of the option from `--source-syntax` to `--workflow`?